src/basics/custom.md

```diff
@@ -0,0 +1,493 @@
+# Custom messages
```

Comment:
I think it would be good to add a single line below that, like: `*since: 0.8.0*`. That would avoid any questions of "why does it not work?!" caused by using an older Sylvia version. I think it should be a pattern we follow in general.

Reply:
I was thinking about it. Agree will add this info